### DISCUSSION TITLE
Flashing zzh: Can't find "CC26X2R1"

### DISCUSSION THREAD

**Discussion by Ursusprimus** [Koenkk/zigbee2mqtt#7028] [Devices]
I finally got the zzh (was stuck in customs for weeks) and now I try to get it running.
The description (https://electrolama.com/radio-docs/#step-3-flash-the-firmware-on-your-stick) is fine but im stuck at: Step 4: Setup zigbee2mqtt or ZHA
It tells to download **CC26X2R1**_20200805.zip from coordinator/Z-Stack_3.x.0/bin/
However, in that folder I can't find such a file. Could someone tell me where it is, please?
![image](https://user-images.githubusercontent.com/79019508/114264252-1ba30e80-99ea-11eb-8a16-9f6dfa0d8879.png)

**Comment by Twit123** [comment]
CC2652R_coordinator_20210120.zip

This is for the standard ZZH and NOT for ZZHP

Simon

**Reply by Ursusprimus** [reply]
Just to make sure (I never flashed a device before and don't want to destroy it): I have a "zzh! CC2652R" and flashed with CC2652R_coordinator_20210120 (Step 3 in description; this should be the first file to flash). But I understand the description in a way that now I have to install another file (CC26X2R1) on top of that. However, I don't find it. 
And the file you mentioned looks like the one I installed already. Does this mean that I don't need to do anything else and the stick is ready?

**Comment by Twit123** [comment]
No, those instructions are outdated, you only need CC2652R_coordinator_20210120.zip and you are done.

Simon

**Comment by Koenkk** [comment]
https://www.zigbee2mqtt.io/information/supported_adapters.html#overview contains a correct direct link to the firmware

**Comment by Ursusprimus** [comment]
Ok, that's clear now, thank you!
Maybe someone could update the description to prevent future misunderstandings...
